examples/fabric-bridge-app/linux/CommissionerControlDelegate.cpp

```diff
@@ -116,7 +116,7 @@ CHIP_ERROR CommissionerControlDelegate::HandleCommissioningApprovalRequest(const
         mLabel.ClearValue();
```

Comment:
Do the example app ZAP files (and all the auto generated code associated with it) need to be removed? Or do we need to keep that?

Reply:
Yes, we still need to retain ZAP for now. The primary objective is to eliminate the Ember-generated code from libCHIP. While there have been discussions about removing ZAP entirely once all Ember-generated code is  removed from libCHIP, but we need to come up with a new scheme to describe the configuration of the app.